eng/Get-TagsDocumentation.ps1

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env pwsh
```

Comment:
This script appears to have been copy and pasted from dotnet-docker.  It looks like it needs to be updated.

Reply:
Fixed